src/lib/annotations/Annotator.js

```diff
@@ -526,6 +585,70 @@ class Annotator extends EventEmitter {
     }
 
     /**
+     * Binds event listeners for draw annotation mode.
+     *
+     * @param {DrawingThread} drawingThread - The drawing thread to bind event listeners to.
+     * @param {HTMLElement} postButtonEl - The HTML element that will save the DrawingThread on click.
+     * @return {void}
+     */
+    bindDrawModeListeners(drawingThread, postButtonEl) {
+        if (drawingThread) {
+            const startCallback = drawingThread.handleStart.bind(drawingThread);
+            const stopCallback = drawingThread.handleStop.bind(drawingThread);
+            const moveCallback = drawingThread.handleMove.bind(drawingThread);
+            /* eslint-disable require-jsdoc */
+            const locationFunction = (event) => this.getLocationFromEvent(event, TYPES.point);
```

Comment:
If we break this out into handlers, some of the objects will get very long. This wrapper is simply for readability.